modules/profile/manifests/modproxy.pp

```diff
@@ -2,13 +2,15 @@
 # 
 #
 #
-class profile::webapp::proxy
-{
-  
-  # config variables for mod_proxy
-  $dns_name=$::profile::webapp::tomcat::dns_name
-  $proxy_conf_file    = '/etc/httpd/conf.d/tomcat.conf'
+class profile::modproxy (
+# config template for mod_proxy
+  $proxy_template     = "profile/proxy.conf.epp",
+){
 
```

Comment:
Do you really need make this profile parametrized?
You don't use parametres in the role.

Reply:
This profile will be used for jenkins and other sources.